Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -109,6 +109,10 @@
     color : "color() * abs(${temperature} - 0.5)"
 });
 
+addStyle('Cos', {
+    color : "color() * cos(${temperature} - Math.PI)"
```

Comment:
As mentioned offline, the styling language does not contain math constants like `PI` yet, so you will need to adjust this example and the tests.